ex4/ex4.js

```diff
@@ -6,3 +6,32 @@
  * toNumber("71") === 71
  * toNumber("a7") === null
  */
+function toNumber(str) {
```

Comment:
Sorry. My bad. This suppose to be for ex4. There are ex3 changes too. Let's undo the ex3 changes and focus on ex4 in this pull request.